tests/test_pmp_iv.py

```diff
@@ -1,4 +1,3 @@
-import os
```

Comment:
Exactamente igual, si tenéis que hacer cambios abrid otro PR con todo lo que no corresponda al PR y quitad ruido del mismo.